docs/modules/ROOT/pages/usage/server.adoc

```diff
@@ -49,6 +53,8 @@ user             16060   0.0  0.0  5078568   2264   ??  S     7:54AM   0:00.00 r
 user             16337   0.0  0.0  5331560   2396   ??  S    23:51PM   0:00.00 rubocop --server /Users/user/src/github.com/rubocop/rubocop-rails
 ```
 
+NOTE: For JRuby and Windows, it might be `rubocop --start-server`.
```

Comment:
What does this note mean?

Reply:
I can reword that as "the process name might be"

Reply:
Ah that makes sense to me now, thanks